django/db/backends/base/schema.py

```diff
@@ -269,7 +269,9 @@ def table_sql(self, model):
         sql = self.sql_create_table % {
             "table": self.quote_name(model._meta.db_table),
             "definition": ", ".join(
-                constraint for constraint in (*column_sqls, *constraints) if constraint
```

Comment:
I moved casting to strings to `constraints` because `column_sqls` are always strings.